_rules/SC1-2-1-media-alternative-audio.md

```diff
@@ -18,17 +18,17 @@ authors:
 
 ### Applicability
 
-The rule applies to any [non-streaming](#non-streaming-media-element) `audio` element that is playing or with a "play button".
+The rule applies to any [non-streaming](#non-streaming-media-element) `audio` element that is playing or has a "play button".  If a play button exists the rule applies if the button is [visible](#visible) or [included in the accessibility tree](#included-in-the-accessibility-tree).
 
-A play button is an interactive element that when activated, plays the audio. The play button must be ]visibile on the page](#visible-on-the-page) or [included in the accessibility tree](#included-in-the-accessibility-tree)
+A play button is an interactive element that when activated, plays the audio. The play button must be [visible](#visible) or [included in the accessibility tree](#included-in-the-accessibility-tree).
```

Comment:
Suggest a definition to avoid inconsistencies.

Reply:
See above. Thank you for the suggestion.